sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -13,8 +16,20 @@ internal static class Statsbeat
 
         private const string StatsBeat_ConnectionString = "<StatsBeat_ConnectionString>";
 
+        private const string AMS_Url = "http://169.254.169.254/metadata/instance/compute?api-version=2017-08-01&format=json";
+
         internal const int AttachStatsBeatInterval = 86400000;
 
+        internal static string s_resourceProviderId;
```

Comment:
Is there a reason for it to be `internal` instead of `private`?

Reply:
was thinking about using it in tests. will change it later if needed.